package.json

```diff
@@ -36,6 +36,7 @@
     "concurrently": "^5.1.0",
     "cross-env": "^7.0.0",
     "cypress": "^4.0.2",
+    "prettier": "^1.19.1",
```

Comment:
"We recommend pinning an exact version of prettier in your package.json as we introduce stylistic changes in patch releases."

Reply:
#classic